frontend/src/Components/Header.tsx

```diff
@@ -0,0 +1,30 @@
+import styled from "styled-components";
```

Comment:
```suggestion
import styled from "@emotion/styled";
```

would look like this then (if u decide using emotion instead)